Source/Scene/Cesium3DTileset.js

```diff
@@ -1237,6 +1239,23 @@ define([
             get : function() {
                 return this._ellipsoid;
             }
+        },
+
+        /**
+         * Returns the `extras` property at the top-level of the tileset JSON, which contains application specific metadata.
+         * Returns `undefined` if `extras` does not exist.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {*}
+         * @readonly
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#specifying-extensions-and-application-specific-extras|Extras in the 3D Tiles specification.}
+         */
+        extras : {
+            get : function() {
+                return this._extras;
+            }
```

Comment:
It might be a good idea to check if the tileset is ready before accessing `extras` and throw an error otherwise. See `asset`:

```
//>>includeStart('debug', pragmas.debug);
if (!this.ready) {
    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
}
//>>includeEnd('debug');
```

Reply:
Only needed for the `Cesium3DTileset` one, not `Cesium3DTile`.